.github/ISSUE_TEMPLATE/blog_submission.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Blog submission request
+about: Submit one blog post to be published in our website
+title: 'blog: '
+labels: 'blog'
+assignees: ''
+
+---
+
+### Blog Metadata
+
+Please provide the following information:
+
+* Title:
+* Author:
+* Date:
+* Short description:
+* Google doc Link with the content:
+* Category
+
+
+### Banner image
+
+Please upload banner image for your post as a suggestion.
+Keep in mind the image should be free or your own image creation.
+In case you don't have any, the team can suggest.
```

Comment:
For the banner, we might suggest to attach a link with the image (so the quality will remain the same).